### PR TITLE
feat: [SEI-3751]: handling license and trial page for SEI

### PR DESCRIPTION
**Pull request by harshilbits**
### Summary
1. Handling license and trial page for SEI

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
- MessageMetadata: `trigger messagecheck`
- GitLeaks: `trigger gitLeaks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### Rebase Instructions:
To rebase your branch onto the latest changes in the target branch, simply use the following command in a comment: `/rebase`

### CODE REVIEW

File: src/modules/75-sei/RouteDestinationsV2.tsx [hunk 1]
```diff
@@ -24,63 +24,63 @@ import { NameSchema } from '@common/utils/Validation'
...
+  )
+}
 
-// const licenseRedirectData: LicenseRedirectProps = {
-//   licenseStateName: LICENSE_STATE_NAMES.SEI_LICENSE_STATE,
-//   startTrialRedirect: () => <TrialRedirect />,
-//   expiredTrialRedirect: RedirectToSubscriptionsFactory(ModuleName.SEI)
-// }
+const licenseRedirectData: LicenseRedirectProps = {
+  licenseStateName: LICENSE_STATE_NAMES.CD_LICENSE_STATE,
```
Comment (sharath-levelops) [inline src/modules/75-sei/RouteDestinationsV2.tsx:60]: Revert back to `SEI_LICENSE_STATE`

Comment (harshilbits) [reply src/modules/75-sei/RouteDestinationsV2.tsx]: yeah...corrected.

**Review by sharath-levelops — APPROVED** [review]

File: src/modules/75-sei/pages/SEITrialPage/images/seiTrial.svg
Comment (Sagar-Rajput7) [inline src/modules/75-sei/pages/SEITrialPage/images/seiTrial.svg:1]: Please see if we can optimize this

**Review by Sagar-Rajput7 — APPROVED** [review]

File: src/modules/75-sei/pages/SEITrialPage/images/seiTrial.svg
Comment (harshilbits) [reply src/modules/75-sei/pages/SEITrialPage/images/seiTrial.svg:1]: Its a svg file , sure will check if any optimisation is needed.

**Review by ajay-levelops — APPROVED** [review]

**Pull request merged**